File: solutions/_705_design_hashset/solution.js
class MyHashSet {
  constructor() {
    this.set = new Set();
  }

  add(key) {
    this.set.add(key);
  }

  remove(key) {
    this.set.delete(key);
  }

  contains(key) {
    return this.set.has(key);
  }
}

module.exports = { MyHashSet };

/*
https://leetcode.com/problems/design-hashset/
*/

File: solutions/_705_design_hashset/solution.py
class MyHashSet:
    def __init__(self):
        self.set = set()

    def add(self, key: int) -> None:
        self.set.add(key)

    def remove(self, key: int) -> None:
        if key in self.set:
            self.set.remove(key)

    def contains(self, key: int) -> bool:
        return key in self.set


# https://leetcode.com/problems/design-hashset/
